stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -413,8 +410,13 @@ function(_add_target_variant_link_flags)
     list(APPEND result "-Wl,-Bsymbolic")
   elseif("${LFLAGS_SDK}" STREQUAL "ANDROID")
     list(APPEND link_libraries "dl" "log")
+    if("${LFLAGS_ARCH}" STREQUAL "armv7")
+      list(APPEND link_libraries "atomic")
+    endif()
     # We need to add the math library, which is linked implicitly by libc++
     list(APPEND result "-lm")
+    list(APPEND result "-rtlib=compiler-rt")
```

Comment:
We can remove this line after the rebranch, as this is the default for Android with clang 13, but the next line is the path in the NDK for `libunwind.a`, so we'll need to pass that since we're linking with the Swift-forked clang.